Procfile

```diff
@@ -1 +1 @@
-web: npm start
+web: npm run build && npm start
```

Comment:
What this means is that whenever a new Dyno is spun up for `underdog-styles` that we will be running `npm run build`.

For example, if the dyno goes to sleep, and then we access the site, we will have to wait for `npm run build` to complete before we'll be able to access it.